datacapturing/src/main/java/de/cyface/datacapturing/DataCapturingService.java

```diff
@@ -321,7 +329,7 @@ public void stop(final @NonNull ShutDownFinishedHandler finishedHandler)
 
                 // We update the {@link MeasurementStatus} here to make sure the {@link Measurement} is also finished
                 // is case the {@link DataCapturingBackgroundService} is already dead.
-                persistenceLayer.getPersistenceBehaviour().updateRecentMeasurement(FINISHED);
+                persistenceLayer.getPersistenceBehaviour().updateStatus(FINISHED);
```

Comment:
True. It made sense from an outer perspective:
- CapturingBehaviour.updateStatus() => because during capturing there is only one "special" measurement. but yeah, I change it back.